app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,31 @@
+class ProfilesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :load_user
+
+  def index
```

Comment:
no need to have this action without any logic. Rails automatically load action/view if there are no declared inside controller. But this controller might change a bit when you change routes (check comments inside routes)